CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+- Copy projects .npmrc file if it exists to functions dist directory (#5235)
```

Comment:
I'd say `Respect .npmrc in backends spun up for web frameworks` or something like that, scope it to frameworks